RNTester/js/CameraRollView.js

```diff
@@ -74,10 +58,9 @@ type Props = $ReadOnly<{|
   renderImage: PhotoIdentifier => React.Node,
 
   /**
-   * imagesPerRow: Number of images to be shown in each row.
+   * A boolean that indicates if we should render large or small images.
    */
-
-  imagesPerRow: number,
```

Comment:
We actually depend on this at Facebook. Does FlatList not support this? I'm not sure if we can merge this change without this feature.

Reply:
No, not easily.. let me refactor this one more time to keep imagesPerRow & resolve the dupe issue properly.